src/styles/components/header.scss

```diff
@@ -4,9 +4,9 @@ $header-height: 100px;
   --logo-color-blue-white: white;
 }
 
-@mixin header-width($prop) {
-  #{$prop}-left: 100px;
-  #{$prop}-right: 100px;
+@mixin header-width($prop: padding, $amount: 100px) {
```

Comment:
This works on the MHI landing page but not on the Roar landing page. We want to center the logo on mobile, no?